Gems/PhysX/Code/Source/EditorRigidBodyComponent.cpp

```diff
@@ -78,11 +78,18 @@ namespace PhysX
                     continue;
                 }
 
+                const AZ::Vector3& assetScale = shapeConfigurationProxy.m_physicsAsset.m_configuration.m_assetScale;
+                const bool isAssetScaleUniform =
+                    AZ::IsClose(assetScale.GetX(), assetScale.GetY()) && AZ::IsClose(assetScale.GetX(), assetScale.GetZ());
```

Comment:
nit: Another IsUniform check (mentioned above)

Reply:
Covered in the previous comment.